index.html

```diff
@@ -176,6 +175,31 @@ <h5 class="categories-title">Skills</h5>
         </div>
     </div>
 </div>
+<form class="contact_forms" action="index.html" method="post">
```

Comment:
You are still missing the action attribute, please add it.

Reply:
okay, so I think I've already posted the, so is not this action="index.html" ???

Reply:
- [ ] The `action` value should be similar to `https://formspree.io/f/{form_id}` as said by the previous reviewer. If this is not clear, learn how to use formspree [here](https://formspree.io/)